src/pages/PatientVitals.js

```diff
@@ -125,14 +134,49 @@ const PatientVitals = ({
   return (
     <div className="wrapper">
       <div className="second-header">
-        <img src={patient_profile} alt="user-profile" />
+        {(progressedPage === NEW_PATIENT_PAGES.patientInfo ||
```

Comment:
please move condition in a constant, i.e.
`const isShowProgressBar = useMemo(() => [NEW_PATIENT_PAGES.patientInfo, NEW_PATIENT_PAGES.covidHistory, NEW_PATIENT_PAGES.preExistingCondition, NEW_PATIENT_PAGES.allergy, NEW_PATIENT_PAGES.medication].includes(progressedPage), [progressedPage])`